src/core/core.controller.js

```diff
@@ -334,9 +334,17 @@ module.exports = function(Chart) {
 			this.tooltip.initialize();
 		},
 
-		update: function(animationDuration, lazy) {
+		update: function(config) {
 			var me = this;
 
+			if (!config || typeof config !== 'object') {
```

Comment:
I can try that. Is there any other place other than `core.controller.tests.js`? Because that isn't testing `_bufferedRender`, nor `render` at the moment.